src/FullBoard.cpp

```diff
@@ -110,6 +110,11 @@ void FullBoard::set_to_move(int tomove) {
 }
 
 int FullBoard::update_board(const int color, const int i) {
+    if (i == FastBoard::PASS) {
+        // No ko
+        return 0;
```

Comment:
Might be a matter of preference but I think it's cleaner to have this logic in the caller.

Reply:
Changed.